contributing/documentation/index.rst

```diff
@@ -1,13 +1,30 @@
 Contributing Documentation
 ==========================
 
-.. toctree::
-    :maxdepth: 2
+Documentation is as important as code for Open Source projects. That's why
+Symfony dedicates a great amount of resources to document new features and to
+keep the rest of the documentation up-to-date.
+
+More than 1,000 developers have contributed to Symfony documentation and we are
```

Comment:
Usually, we avoid the first person perspective. But here it matches quite well IMO.